dis_snek/models/discord/guild.py

```diff
@@ -312,8 +312,17 @@ async def get_owner(self) -> "models.Member":
         # maybe precache owner instead of using `get_owner`
         return await self._client.cache.get_member(self.id, self._owner_id)
 
-    def is_owner(self, member: "models.Member") -> bool:
-        return self._owner_id == member.id
+    def is_owner(self, user_id: Snowflake_Type) -> bool:
```

Comment:
You should Union `models.Member` here, also not sure about renaming the parameter to user_id, since the main idea is whether a member in that guild is the owner.

Reply:
As per definition: `Snowflake_Type = Union[int, str, "SnowflakeObject", SupportsInt]` you don't need to union Snowflake_Type with Member or anything, it's already included
But I agree about renaming back to `user`